src/test/java/com/jcabi/log/UnitTestAppender.java

```diff
@@ -60,11 +70,13 @@ public void activateOptions() {
     }
 
     /**
-     * Return the used OutputStream.
-     * @return OutputStream where the logs are written
+     * Return the logged messages.
+     * @return String logs
+     * @throws UnsupportedEncodingException If the bytes cannot be converted
+     *  to String using UTF-8
      */
-    public ByteArrayOutputStream output() {
-        return this.logs;
+    public String output() throws UnsupportedEncodingException {
+        return new String(this.logs.toByteArray(), "UTF-8");
```

Comment:
@mkordas If I simply made ``new String(this.logs.toByteArray())`` like in the original tests, PMD complains about instantiating Strings... it did not complain in the tests.